mod/wall_upload.php

```diff
@@ -119,6 +119,7 @@ function wall_upload_post(&$a, $desktopmode = true) {
 	$maximagesize = get_config('system','maximagesize');
 
 	if(($maximagesize) && ($filesize > $maximagesize)) {
+		logger("Image exceeds size limit of ".$maximagesize);
 		echo  sprintf( t('Image exceeds size limit of %s'), formatBytes($maximagesize)) . EOL;
```

Comment:
I just stumbled over it, most likely as I've used various echos and var_dumps for the curweather addon. If it's for debugging and you need it make a logger call from it I'd say.